packages/client/src/pages/select_account/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+
+import {
+  useFirestoreSubscribe,
+  useUpdateSubscription,
+} from "@eisbuk/react-redux-firebase-firestore";
+import { OrgSubCollection } from "@eisbuk/shared";
+import { Routes } from "@eisbuk/shared/ui";
+import { LayoutContent } from "@eisbuk/ui";
+
+import Layout from "@/controllers/Layout";
+
+import ErrorBoundary from "@/components/atoms/ErrorBoundary";
+
+import { getOrganization } from "@/lib/getters";
+
+import { getAllSecretKeys } from "@/store/selectors/auth";
+import { getOtherBookingsAccounts } from "@/store/selectors/bookings";
+
+const SelectAccount: React.FC = () => {
+  const secretKeys = useSelector(getAllSecretKeys) || [];
+  const accounts = useSelector(getOtherBookingsAccounts(""));
```

Comment:
Ok, I did this, but it should actually never happen:
- customers with fewer than 2 secret keys wouldn't (at least shouldn't) ever reach the page
- there should be the same number of accounts as secret keys

In case this redirect happens, that indicates there's something broken in the code or the db. It could happen, but the question is: if we're watching out for that (unlikely) scenario, should we also add some way of reporting the mishap to Sentry 🤷